scripts/release

```diff
@@ -12,7 +13,7 @@ rescue StandardError => e
 end
 
 channel = ENV.fetch("FOREM_RELEASE_CHANNEL", "stable")
-date = ENV.fetch("FOREM_RELEASE_DATE", Date.today.to_s).delete("-")
+date = ENV.fetch("FOREM_RELEASE_DATE", Time.zone.today.to_s).delete("-")
```

Comment:
Ok, I merged back main and changed back the code. Since you already ignore two cops in the configs I'll also remove the explicit disabling and reenabling for `Style/TopLevelMethodDefinition`.